### PR TITLE
Start using PSR-2

### PR DESCRIPTION
**Pull request by thewilkybarkid**

### CODE REVIEW

File: tests/cases/classes/method-declaration [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+<?php
+
+namespace Vendor;
+
+class Foo
+{
+    public function bar  ( $baz  ,&$qux,  bool$quux, ? string  $quuz    =null   ,  string...$corge   ):
+    void
+    {
+    }
```
Comment (thewilkybarkid) [inline tests/cases/classes/method-declaration]: Can't include an `abstract` example yet, waiting for https://github.com/squizlabs/PHP_CodeSniffer/pull/2148 to be merged/released.

File: src/Libero/ruleset.xml [hunk 2]
```diff
@@ -5,8 +5,25 @@
...
-    <rule ref="PSR1"/>
+    <rule ref="PSR2"/>
 
+    <rule ref="SlevomatCodingStandard.TypeHints.ParameterTypeHintSpacing"/>
+    <rule ref="SlevomatCodingStandard.TypeHints.ReturnTypeHintSpacing">
+        <properties>
+            <property name="spacesCountBeforeColon" value="1"/>
+        </properties>
+    </rule>
     <rule ref="Squiz.Classes.ClassFileName"/>
```
Comment (thewilkybarkid) [inline src/Libero/ruleset.xml]: No example for `trait`s yet, waiting for https://github.com/squizlabs/PHP_CodeSniffer/pull/2140 to the merged/released.

File: tests/cases/php/separate-use [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---DESCRIPTION---
+One use statement per declaration
+---CONTENTS---
+<?php
+
+namespace Vendor;
+
+use Foo, Bar;
```
Comment (thewilkybarkid) [inline tests/cases/php/separate-use]: No `function` and `const` examples yet, waiting for https://github.com/squizlabs/PHP_CodeSniffer/pull/2143 to be released.

**Comment by thewilkybarkid** [comment]
(Quick reviews would be useful, but will merge later this week otherwise.)

File: tests/cases/php/line-length [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---DESCRIPTION---
+Lines must not exceed 120 characters
+---CONTENTS---
+<?php
+
+$foo = '890123456789001234567890012345678900123456789001234567890012345678900123456789001234567890012345678900123456789';
```
Comment (stephenwf) [inline tests/cases/php/line-length]: Is there a fixed out there for splitting this across more than one line?

File: tests/cases/php/separate-use (hunk 3 above)
Comment (stephenwf) [reply tests/cases/php/separate-use]: How does this deal with PHP 7s grouped statements 
```php
use some\namespace\{ClassA, ClassB, ClassC as C};
```

Comment (thewilkybarkid) [reply tests/cases/php/separate-use]: 5941c0cb6d58a678f09fe8f4a6ebe39740b4b413

Comment (stephenwf) [reply tests/cases/php/separate-use]: ah nice it does it for the functions and consts too 👍

**Pull request merged**